backend/src/controllers/clientCourses/sendAssessmentResult.ts

```diff
@@ -0,0 +1,27 @@
+import { NextFunction, Request, Response } from 'express';
+
+import CourseStatus from 'enums/coursesEnums';
+
+const sendAssessmentResult = async (
+  req: Request,
+  res: Response<
+    string,
+    {
+      status?: CourseStatus;
+    }
+  >,
+  next: NextFunction,
+) => {
+  try {
+    const { status } = res.locals;
+    res.json(
```

Comment:
don't write ternary in res.json() write for that own function

Reply:
file deleted